### PR TITLE
Fixed #4409 : Dark Mode support for Confetti

### LINKED ISSUE #4409: Dark Mode support for Confetti
As per [this](https://github.com/oppia/oppia-android/pull/4404#issuecomment-1168046385) comment we should see if confetti requires dark-mode support or not.

This issue still need discussion to finalise the decision.

**Where is this confetti shown?**
Whenever user answers correctly in Question/Exploration player the confetti is show as below images.
And whenever the exploration / question player is finished completely by answering correctly the confetti is again shown but in a different way.

### PR DESCRIPTION
**Pull request by MohitGupta121**
## Explanation

Fixed #4409  : Dark Mode support for Confetti

|Default|Dark Mode|
|:----------------------------------------:|:-----------------------------------------:|
|  | Screenshot_2022-05-31-12-23-27-332_com android camera |

## Essential Checklist

- [x] The PR title and explanation each start with "Fix #bugnum: " (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] Any changes to [scripts/assets](https://github.com/oppia/oppia-android/tree/develop/scripts/assets) files have their rationale included in the PR explanation.
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary code changes from Android Studio ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#undo-unnecessary-changes)).
- [x] The PR is made from a branch that's **not** called "develop" and is up-to-date with "develop".
- [x] The PR is **assigned** to the appropriate reviewers ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#clarification-regarding-assignees-and-reviewers-section)).

## For UI-specific PRs only

If your PR includes UI-related changes, then:
- Add screenshots for portrait/landscape for both a tablet & phone of the before & after UI changes
- For the screenshots above, include both English and pseudo-localized (RTL) screenshots (see [RTL guide](https://github.com/oppia/oppia-android/wiki/RTL-Guidelines))
- Add a video showing the full UX flow with a screen reader enabled (see [accessibility guide](https://github.com/oppia/oppia-android/wiki/Accessibility-(A11y)-Guide))
- Add a screenshot demonstrating that you ran affected Espresso tests locally & that they're passing

### CODE REVIEW

**Review by rt4914 — APPROVED** [review]
LGTM, Thanks.

File: app/src/main/res/values/color_defs.xml [hunk 1]
```diff
@@ -74,4 +74,9 @@
   <color name="color_def_stroke_silver">#777777</color>
   <color name="color_def_drag_drop_single_item_gradient_center">#F7F7F7</color>
   <color name="color_def_drag_drop_single_item_gradient_end">#DCDCDC</color>
+  <color name="color_def_confetti_rubby_red">#FF1414</color>
```
Comment (BenHenning) [inline app/src/main/res/values/color_defs.xml]:
```suggestion
  <color name="color_def_confetti_ruby_red">#FF1414</color>
```

**Review by BenHenning — COMMENTED** [review]
Thanks @MohitGupta121! I think this looks really nice! Just had one nit on one of the color's names, otherwise the PR LGTM.

**Comment by MohitGupta121** [comment]
@BenHenning I commits your suggestion changes. Can you please review it now.

**Review by BenHenning — APPROVED** [review]
Thanks @MohitGupta121! This LGTM. However, please don't resolve reviewers' conversation threads. To indicate something is done, you just need to reply with "Done" and then the reviewer will close the thread once they confirm that it's been addressed as expected (otherwise it's hard to track requested changes across review passes). Note that this is covered in https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR so I suggest taking another pass on that just make sure that you're familiar with the team's PR process. :)

**Comment by MohitGupta121** [comment]
> 

Oh really sorry actually when I commit new changes it's automatically mark as resolved. As Rajat, in past also told me the same but don't know it's automatically resolved. I take care next time carefully.

**Comment by BenHenning** [comment]
> Oh really sorry actually when I commit new changes it's automatically mark as resolved. As Rajat, in past also told me the same but don't know it's automatically resolved. I take care next time carefully.

Hmm @MohitGupta121 I've never run into this before. Can you reach out to me if you notice again? I don't think GitHub conversation threads are ever supposed to automatically resolve.

**Comment by MohitGupta121** [comment]
> > Oh really sorry actually when I commit new changes it's automatically mark as resolved. As Rajat, in past also told me the same but don't know it's automatically resolved. I take care next time carefully.
> 
> Hmm @MohitGupta121 I've never run into this before. Can you reach out to me if you notice again? I don't think GitHub conversation threads are ever supposed to automatically resolve.

Sure if I notice it again I let you know about that. But I think I click on the changes suggest and commit from that. Due to this I think that happens as I remember.

**Comment by BenHenning** [comment]
Ah @MohitGupta121 yes. I forgot about that case. Accepting a suggestion will auto-resolve the conversation thread. I suggest, in those cases, to just reopen the thread and reply to it like you would any other comment.

**Comment by MohitGupta121** [comment]
> Ah @MohitGupta121 yes. I forgot about that case. Accepting a suggestion will auto-resolve the conversation thread. I suggest, in those cases, to just reopen the thread and reply to it like you would any other comment.

oh okay, take care of thread opening next time.

**Pull request merged**